src/alto/server/estimate_throughput.py

```diff
@@ -0,0 +1,112 @@
+"""
+estimate_throughput.py
+
+A Flask app that runs a basic ALTO server.
+
+IMPORTANT NOTES:
+* This script assumes that the configuration files are located at
+	"input/g2.conf" and "output/input_routing.conf". If this is not the case,
+	then change the variables g2fp_str and infp_str
+* This script estimates RTT by summing up the delays of each link. This is
+	not accurate!!! But, since taking into account queuing dynamics is
+	too difficult, the script does this instead.
+"""
+
+import json
+import numpy as np
+
+NETWORK_TYPE = "G2_MININET"
+"""The format in which the network is configured.
+
+Currently, only `G2_MININET` is supported.
+"""
+
+SOLVER = "jensen"

```

Comment:
Maybe "num" is a better name for the solver, as this solver uses the "Network Utility Maximization (NUM)" model to do the prediction.

Reply:
Never see the name is changed. Re-opened this conversation.